app/scripts/services/navigate.js

```diff
@@ -82,6 +82,22 @@ angular.module("openshiftConsole")
         $location.path(this.membershipURL(projectName));
       },
 
+      catalogURL: function(project) {
+        var projectName = angular.isString(project) ? project : _.get(project, 'metadata.name');
+        if (!projectName) {
+          return './';
+        }
+
+        return "project/" + encodeURIComponent(projectName) + "/catalog";
+      },
+
+      toProjectCatalog: function(projectName, search) {
```

Comment:
nit: You might change the parameter name to `project` since it can be an object, too. (Probably worth a comment as well.)

Reply:
renamed and added comments